### PR TITLE
Parser

### LINKED ISSUE #5: Ideas scrapper

### PR DESCRIPTION
**Pull request by MrPaschenko**
Closes #5

### CODE REVIEW

File: src/parser/index.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const fs = require('fs');
+const cheerio = require('cheerio');
+// const chalk = require('chalk');
```
Comment (akaeyuhi) [inline src/parser/index.js:5]: Delete commented lines pls

File: src/parser/index.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            .html();
+
+          let ingredient = name;
+          mass ? ingredient = name + ' - ' + mass : ingredient = name;
+
+          ingredients.push(ingredient);
+        });
+      const products = ingredients.join('\n');
+
+      // eslint-disable-next-line camelcase
```
Comment (akaeyuhi) [inline src/parser/index.js:58]: Disabled eslint

File: src/parser/index.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      const category_id = $1('.article-breadcrumbs')
```
Comment (akaeyuhi) [inline src/parser/index.js:59]: Use camelcase for variables

File: src/parser/index.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+const fs = require('fs');
+const cheerio = require('cheerio');
+// const chalk = require('chalk');
+const pupp = require('./helpers/puppeteer');
+const common = require('./helpers/common');
+// const listItemsHandler = require('./handlers/listItemsHandler');
+const SITE = 'https://www.povarenok.ru/recipes/';
+const pages = 1;
+
+(async function main() {
```
Comment (akaeyuhi) [inline src/parser/index.js:12]: Big undecomposed function

**Review by akaeyuhi — CHANGES_REQUESTED** [review]
Use camelcase, decompose main function

File: src/parser/index.js (hunk 3 above)
Comment (MrPaschenko) [reply src/parser/index.js:59]: `snake_case` is required from db team

File: src/parser/index.js (hunk 2 above)
Comment (MrPaschenko) [reply src/parser/index.js:58]: Used for `camel_case`

File: src/parser/index.js (hunk 1 above)
Comment (MrPaschenko) [reply src/parser/index.js]: Done

**Comment by yanu1ya** [comment]
пітца

**Review by akaeyuhi — APPROVED** [review]

**Pull request merged**